src/OctoshiftCLI.Tests/Octoshift/Commands/AbortMigration/AbortMirationCommandHandlerTests.cs

```diff
@@ -0,0 +1,65 @@
+using System.Threading.Tasks;
+using Moq;
+using OctoshiftCLI.Commands.AbortMigration;
+using OctoshiftCLI.Services;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.Octoshift.Commands.AbortMigration;
+
+public class AbortMigrationCommandHandlerTests
+{
+    private readonly Mock<GithubApi> _mockGithubApi = TestHelpers.CreateMock<GithubApi>();
+    private readonly Mock<OctoLogger> _mockOctoLogger = TestHelpers.CreateMock<OctoLogger>();
+
+    private readonly WarningsCountLogger _warningsCountLogger;
+    private readonly AbortMigrationCommandHandler _handler;
+
+    private const string REPO_MIGRATION_ID = "RM_MIGRATION_ID";
+
+    public AbortMigrationCommandHandlerTests()
+    {
+        _warningsCountLogger = new WarningsCountLogger(_mockOctoLogger.Object);
+        _handler = new AbortMigrationCommandHandler(_mockOctoLogger.Object, _mockGithubApi.Object);
+
+        TestHelpers.SetCliContext();
```

Comment:
no need to set the context
```suggestion
        TestHelpers.SetCliContext();
```